### ISSUE TITLE
Forced-look after copyover has issues

### ISSUE THREAD

**Issue by luciensadi** [luciensadi/AwakeMUD#21]
After a copyover, your character is forced to LOOK the moment your descriptor is brought online. This creates a potential knowledge gap where another player in the room could load after you look and thus appear without you knowing they're there. Solutions for this include either doing a second loop and forcing a look after everyone has loaded, or alternatively printing a message when someone restores from copyover (potentially spammy).

Also, screenreader players probably do not appreciate being forced to look at the full description. Investigate ways to opt out of this behavior.

**Comment by jalles76** [comment]
I'd go for the spammy option and skip the look. Anyone receiving a "player has recovered from copyover" message would almost automatically issue a look command.

**Comment by luciensadi** [comment]
Pushed fix for out-of-order looking; now all PCs 'look' only after the load is complete. Still need to investigate screenreader compatibility, but I'll open a new issue for that since I want to make it into a full project.
